packages/quantic/force-app/main/default/lwc/quanticTab/quanticTab.js

```diff
@@ -44,8 +44,23 @@ export default class QuanticTab extends LightningElement {
    * Whether to clear the filters when the active tab changes.
    * @api
    * @type {boolean}
+   * @defaultValue `false`
    */
-  @api clearFiltersOnTabChange = false;
+  @api get clearFiltersOnTabChange() {
+    return this._clearFiltersOnTabChange;
+  }
+  set clearFiltersOnTabChange(value) {
+    // Explicit boolean conversion to handle string values like "false"
+    // Handle string values like "true" or "false"
+    if (typeof value === 'string') {
+      // @ts-ignore
+      this._clearFiltersOnTabChange = value.toLowerCase() === 'true';
+    } else {
+      // Handle boolean, null, undefined, and other types
+      this._clearFiltersOnTabChange = Boolean(value);
+    }
+  }
```

Comment:
I was thinking on a more simpler approach 🤔 what do you think?
```suggestion
  set clearFiltersOnTabChange(value) {
    this._clearFiltersOnTabChange =  value === true || value === 'true';
  }
```

Reply:
I simplified it as I agree it might have been a bit overkill haha but I do think we should keep the string conversion to lowercase in the off chance the user puts `False` or `FALSE`